packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -84,10 +90,12 @@ class VMInfoList extends StatelessWidget {
                           ),
                           const SizedBox(width: denseSpacing),
                           Flexible(
-                            child: SelectableText(
-                              row.value?.toString() ?? '--',
-                              style: theme.fixedFontStyle,
-                            ),
+                            child: row.value is Widget
+                                ? row.value
+                                : SelectableText(
+                                    row.value?.toString() ?? '--',
```

Comment:
We should make `'--'` a constant at some point.

Reply:
Yes, it is used in a few places...